### PR TITLE
Task dev 3

### PR DESCRIPTION
**Pull request by EvgenyKisel**

### CODE REVIEW

File: DEV-3/FibonacciNumbers/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bin/
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/.gitignore]: .gitignore is global, for master, relocate to main git folder

File: DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                            fibNumb2 = fibNumbN;
+                            fibNumbN = fibNumb1 + fibNumb2;
+                            if (number == fibNumbN || number == 1)
+                            {
+                                checkFibonacci = true;
+                            }
+                        }
+                        if (checkFibonacci)
+                        {
+                            Console.WriteLine("This is the number of Fibonacci.");
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: Fibonnacci number, not "number of Fibonacci"

File: DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                        int fibNumb1 = 1, fibNumb2 = 1, fibNumbN = 1;
+                        bool checkFibonacci = false;
+                        for (int i = 0; i < number; i++)
+                        {
+                            fibNumb1 = fibNumb2;
+                            fibNumb2 = fibNumbN;
+                            fibNumbN = fibNumb1 + fibNumb2;
+                            if (number == fibNumbN || number == 1)
+                            {
+                                checkFibonacci = true;
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: not optimal decision, no cycle break

File: DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    int number = int.Parse(Console.ReadLine());
+                    if (number < 0)
+                    {
+                        Console.WriteLine("It is a negative number. Try again");
+                        continue;
+                    }
+                    else
+                    {
+                        int fibNumb1 = 1, fibNumb2 = 1, fibNumbN = 1;
+                        bool checkFibonacci = false;
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: Fibonacci number calculation has to be in a separate method (and class)

File: DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs [hunk 5]
```diff
@@ -0,0 +1,55 @@
+﻿using System;
+
+namespace FibonacciNumbers
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: Never use while(true), think about more suitable cycle

File: DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs [hunk 6]
```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+namespace FibonacciNumbers
+{
+    class Program
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: Class name is not perfect ))

Comment (EvgenyKisel) [reply DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: (((

Comment (IgorKheidorov) [reply DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: Give normal name to class

Comment (EvgenyKisel) [reply DEV-3/FibonacciNumbers/FibonacciNumbers/Program.cs]: have already done this) but doesn't know how it's would be now

File: DEV-3/FibonacciNumbers/FibonacciNumbers.sln [hunk 7]
```diff
@@ -0,0 +1,22 @@
+﻿
```
Comment (IgorKheidorov) [inline DEV-3/FibonacciNumbers/FibonacciNumbers.sln]: Solution has to be one for all tasks!!!

**Review by IgorKheidorov — APPROVED** [review]
Not bad, hope for green cell in future)

**Pull request merged**